proknow-sdk/Scorecard/ScorecardTemplateItemJsonConverter.cs

```diff
@@ -96,6 +102,13 @@ public override void Write(Utf8JsonWriter writer, ScorecardTemplateItem scorecar
                 writer.WriteStringValue(scorecardTemplateItem.Name);
             }
 
+            // workspaceid
+            if (scorecardTemplateItem.WorkspaceID != null) 
+            {
+                writer.WritePropertyName(_workspaceID);
+                writer.WriteStringValue(scorecardTemplateItem.WorkspaceID);
```

Comment:
Add test for this in ScorecardTemplateItemJsonConverterTest maybe like WriteTestWorkspaceAsync()?

Reply:
Added test